Meta/CMake/serenity_toolchain.cmake

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Same comment as in `Superbuild/CMakeLists.txt`: Any reason why a proper toolchain is not used instead?